### DISCUSSION TITLE
Ctrlr Architecture Diagrams/Documentation

### DISCUSSION THREAD

**Discussion by unityconstruct** [RomanKubiak/ctrlr#606] [Q&A] [answered]
Class Diagrams

**Comment by unityconstruct** [comment, marked as answer]
### High level Class Diagram: CtrlrPanel, CtrlrModulator, CtrlrLua
![ctrlrx-shard-code-classdiagram](https://github.com/RomanKubiak/ctrlr/assets/40268430/078cb740-0992-48d6-836a-5b0073ce8136)

**Comment by unityconstruct** [comment]
### Methods Class Diagram: Lua-Related Classes
![ctrlr-lua](https://github.com/RomanKubiak/ctrlr/assets/40268430/962b3967-dc92-47e4-ba12-81f2b98f4234)

**Comment by unityconstruct** [comment]
# CtrlrComponent Methods

![CtrlrComponent](https://github.com/RomanKubiak/ctrlr/assets/40268430/c48fa9b0-e8fc-40fa-b5e0-c38577ef23e5)

**Reply by damiensellier** [reply]
Congrats @unityconstruct ! You did a great job of documentation. Merci
